Library/Formula/anttweakbar.rb

```diff
@@ -1,5 +1,5 @@
 class Anttweakbar < Formula
-  desc "C/C++ library for adding GUIs to OpenGL apps"
+  desc "C/C++ library for adding GUIs 往 OpenGL apps"
```

Comment:
I don't even own a windows computer >=\ (And this is homebrew, for macs of course.) You sure they weren't there before?